docs/source/projects.rst

```diff
@@ -212,7 +212,8 @@ Docker container environment
   
   .. code-block:: yaml
 
-    docker_env: mlflow-docker-example-environment
```

Comment:
Is this a breaking change or is it just a problem with the docs. It seems like before this PR it was already looking at the `image` field.

Reply:
Yes, the docs were wrong.